src/core/PathUtil.cpp

```diff
@@ -188,4 +188,13 @@ namespace lmms::PathUtil
 		return basePrefix(shortestBase) + relativeOrAbsolute(absolutePath, shortestBase);
 	}
 
+	std::filesystem::path LMMS_EXPORT qStringToPath(const QString& path)
+	{
+#ifdef LMMS_BUILD_WIN32
+		return path.toStdWString().c_str();
+#else
+		return path.toStdString().c_str();
```

Comment:
I remember there are some QT functions that convert a `QString` to `std::filesystem::path` in QT 6, but I believe lmms is using QT 5 currently. If this is the case, please leave a comment.

I think this `path.toStdString()` should be static cast to `std::filesystem::path`, and I believe `.c_str()` can be removed.

Reply:
I prefer this, but maybe I can add TODO comments if Qt 6 already provides these functions. We might also want to do this effort in a separate PR, so maybe I'll just use `QString` temporarily.

Reply:
You can use `std::basic_string` in the constructor of `std::filesystem::path`, you don't need to use `.c_str()`.

So I think ideally this line should look like this:
```suggestion
		return std::filesystem::path(path.toStdString());
```

I's fine if you don't follow this suggestion, but make sure to leave a comment about `std::filesystem::path QFileInfo::filesystemFilePath()`